src/rpc/RPCHelpers.cpp

```diff
@@ -1602,6 +1602,16 @@ traverseTransactions(
             cursor = {maxIndex, INT32_MAX};
     }
 
+    if (request.contains(JS(ledger_index_min)) &&
+        request.contains(JS(ledger_index_max)))
+    {
+        auto& max = request.at(JS(ledger_index_max));
+        auto& min = request.at(JS(ledger_index_min));
+
+        if (max.as_int64() < min.as_int64())
+            return Status{Error::rpcLGR_IDXS_INVALID, "lgrIdxsInvalid"};
+    }
```

Comment:
This is correct, but inefficient, since we already parsed the min and the max from the request higher up in the function. Let's avoid parsing the same value twice, and just use some of the existing variables.

Reply:
@cjcobb23 resolved by moving auto max and min from preexisting occurences outside by one order of precedence